spec/api.yaml

```diff
@@ -124,6 +140,39 @@ paths:
               schema:
                 $ref: '#/components/schemas/User'
 
+  /user/edit-profile:
```

Comment:
Endpoint can just be called `/user/profile`, semantically we know edit request by the PUT method